content/en/docs/concepts/overview/components.md

```diff
@@ -14,140 +13,73 @@ card:
 ---
 
 <!-- overview -->
-When you deploy Kubernetes, you get a cluster.
-{{< glossary_definition term_id="cluster" length="all" prepend="A Kubernetes cluster consists of">}}
 
-This document outlines the various components you need to have for
-a complete and working Kubernetes cluster.
+This page provides a high-level overview of the essential components that make up a Kubernetes cluster.
 
 {{< figure src="/images/docs/components-of-kubernetes.svg" alt="Components of Kubernetes" caption="The components of a Kubernetes cluster" class="diagram-large" clicktozoom="true" >}}
 
 <!-- body -->
-## Control Plane Components
 
-The control plane's components make global decisions about the cluster (for example, scheduling),
-as well as detecting and responding to cluster events (for example, starting up a new
-{{< glossary_tooltip text="pod" term_id="pod">}} when a Deployment's
-`{{< glossary_tooltip text="replicas" term_id="replica" >}}` field is unsatisfied).
+## Core Components
 
-Control plane components can be run on any machine in the cluster. However,
-for simplicity, setup scripts typically start all control plane components on
-the same machine, and do not run user containers on this machine. See
-[Creating Highly Available clusters with kubeadm](/docs/setup/production-environment/tools/kubeadm/high-availability/)
-for an example control plane setup that runs across multiple machines.
+A Kubernetes cluster consists of a control plane and one or more worker nodes. Here's a brief overview of the main components:
 
-### kube-apiserver
+### Control Plane Components
 
-{{< glossary_definition term_id="kube-apiserver" length="all" >}}
+Manage the overall state of the cluster:
 
-### etcd
+<dl>
+  <dt>kube-apiserver</dt>
+  <dd>The API server that exposes the Kubernetes API</dd>
 
-{{< glossary_definition term_id="etcd" length="all" >}}
+  <dt>etcd</dt>
+  <dd>Consistent and highly-available key value store for all cluster data</dd>
 
-### kube-scheduler
+  <dt>kube-scheduler</dt>
+  <dd>Assigns Pods to Nodes</dd>
 
-{{< glossary_definition term_id="kube-scheduler" length="all" >}}
+  <dt>kube-controller-manager</dt>
+  <dd>Runs controller processes</dd>
 
-### kube-controller-manager
+  <dt>cloud-controller-manager</dt>
+  <dd>Integrates with underlying cloud providers</dd>
+</dl>
```

Comment:
/docs/concepts/architecture/#kube-apiserver
```suggestion
[kube-apiserver](/docs/concepts/architecture/#kube-apiserver)
: The core component server that exposes the Kubernetes HTTP API

[etcd](/docs/concepts/architecture/#etcd)
: Consistent and highly-available key value store for all API server data

[kube-scheduler](/docs/concepts/architecture/#kube-scheduler)
: Looks for Pods not yet bound to a node, and assigns each Pod to a suitable node.

[kube-controller-manager](/docs/concepts/architecture/#kube-controller-manager)
: Runs {{< glossary_tooltip text="controllers" term_id="controller" >}} to implement Kubernetes API behavior.

[cloud-controller-manager](/docs/concepts/architecture/#cloud-controller-manager) (optional)
: Integrates with underlying cloud provider(s)
```

Reply:
thanks for the tip @sftim, I think there might be other sections that need converting too, I'll fix them in a separate commit!